apps/mobile/src/navigations/root/root.tsx

```diff
@@ -16,10 +17,12 @@ const screenOptions: NativeStackNavigationOptions = {
 
 const Root: React.FC = () => {
 	return (
-		<NativeStack.Navigator screenOptions={screenOptions}>
-			<NativeStack.Screen component={Auth} name={RootScreenName.SIGN_IN} />
-			<NativeStack.Screen component={Auth} name={RootScreenName.SIGN_UP} />
-		</NativeStack.Navigator>
+		<SafeAreaScreenWrapper>
```

Comment:
There is no need to wrap the Navigator with it. It is supposed to be used on the exact screen